packages/client-sdk/src/iframe/position.ts

```diff
@@ -71,14 +106,143 @@ function getMobileLocation(
 }
 
 /**
- * Full remote wallet position configuration, any missing parts have default values.
+ * Full wallet position configuration, any missing parts have default values.
  */
 export function getWalletPosition(
   desktop?: Partial<WalletButtonDesktopPosition> | undefined,
   mobile?: Partial<WalletButtonMobilePosition> | undefined,
 ): WalletButtonPosition {
   return {
-    desktop: getDesktopLocation(desktop),
-    mobile: getMobileLocation(mobile),
+    desktop: getDesktopConfig(desktop),
+    mobile: getMobileConfig(mobile),
   };
 }
+
+/**
+ * Normalize shift to ensure valid value.
+ */
+function normalizeShift(value: number, max: number) {
+  if (value < 0) {
+    return 0;
+  }
+  if (value > max) {
+    return max;
+  }
+  return value;
+}
+
+/**
+ * Transform number value to css pixel representation.
+ */
+export function toPixel(n?: number): string {
+  // empty string is not set
+  return n == null ? "" : `${n}px`;
```

Comment:
```suggestion
  return n === null ? "" : `${n}px`;
```

can it only be null? no undefined right?

Reply:
Hm, it can be undefined if not passed in right? But now I need to re-read best practices here.